notifier/src/lib.rs

```diff
@@ -180,6 +190,15 @@ impl Notifier {
                     }
                 }
 
+                NotificationType::PagerDuty(routing_key) => {
+                    let data = json!({"payload":{"summary":msg,"source":"solana-watchtower","severity":"critical",},"routing_key":routing_key,"event_action":"trigger"});
```

Comment:
```suggestion
                    let data = json!({"payload":{"summary":msg,"source":"solana-watchtower","severity":"critical"},"routing_key":routing_key,"event_action":"trigger"});
```

Reply:
ah thanks. I saw a CI failure on that line initially and thought it was because I had broken it across multiple lines, so pushed it all together into one line, totally missed that trailing comma!